editor/editor_themes.cpp

```diff
@@ -179,17 +197,14 @@ void editor_register_and_generate_icons(Ref<Theme> p_theme, bool dark_theme = tr
 #endif
 }
 
-#define HIGHLIGHT_COLOR_FONT highlight_color.linear_interpolate(dark_theme ? Color(1, 1, 1, 1) : Color(0, 0, 0, 1), 0.5)
-#define HIGHLIGHT_COLOR_BG highlight_color.linear_interpolate(dark_theme ? Color(0, 0, 0, 1) : Color(1, 1, 1, 1), 0.5)
```

Comment:
really good you got rid of those macros. It made no sense to have some macro colors and the rest as Variables.